lib/data_model/register_model.dart

```diff
@@ -4,7 +4,7 @@ class RegisterModel {
 
   Map<String, dynamic> toJSON() {
     return {
-      "id": 0,
+      "id": 1,
```

Comment:
No need to pass the id when registering as well. Was this causing an issue?

Reply:
Oh for this, the registration wasn't working so I tried to fix things but that didn't work still